hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -174,6 +174,17 @@ public final class Hedera implements SwirldMain {
     private HandleThrottleAccumulator handleThrottling;
     private MonoMultiplierSources monoMultiplierSources;
 
+    /**
+     * The application name from the platform's perspective. This is currently locked in at the old main class name and
+     * requires data migration to change.
+     */
+    public static final String APP_NAME = "com.hedera.services.ServicesMain";
+
+    /**
+     * The swirld name. Currently there is only one swirld.
+     */
+    public static final String SWIRLD_NAME = "123";
```

Comment:
I do not understand what the swirldName is. Documentation in PlatformBuilder does not help. Independent of that I assume that "123" is not a good value for our real application

Reply:
It's currently an unused feature, more or less. But it's something that is currently used in file system paths, so we have to carry it forward until we do a data migration. Right now all hedera deployments use the swirld name "123". This is data that currently gets passed in via the `config.txt` file. Storing it as a constant that gets passed into the constructor will eliminate that dependency.

Reply:
Does it makes sense to create an issue for that to have a more meaningful name in future?

Reply:
Created issue: https://github.com/hashgraph/hedera-services/issues/9029